storage/interfaces.go

```diff
@@ -182,7 +175,6 @@ type TransactionalRepositoryDecorator func(TransactionalRepository) (Transaction
 //go:generate counterfeiter . Storage
 type Storage interface {
 	OpenCloser
-	Pinger
```

Comment:
This works just because the storage implementation is hardcoded in `sm.New` to use `postgres.Storage`. Assuming it is not, this should be reverted.